src/attribute.rs

```diff
@@ -22,9 +20,41 @@ impl Attribute {
     name.lexeme().parse().ok()
   }
 
-  pub(crate) fn to_str(self) -> &'static str {
+  pub(crate) fn to_str(&self) -> &'static str {
     self.into()
   }
+
+  /// Returns a range from the min to max expected arguments of a given attribute
+  pub(crate) fn expect_args(&self) -> Range<usize> {
+    use Attribute::*;
+
+    match self {
+      Confirm(_) => 1..2,
```

Comment:
Should this be `0..=1`, since confirm takes 0 or 1 argument?

Reply:
Well is `[confirm()]` valid syntax?

Reply:
Ah, yeah, I see what you're saying. You're right, I think `confirm()` shouldn't be valid syntax.